src/vectorchord/usage/measure-recall.md

```diff
@@ -22,12 +30,114 @@ SELECT vchordrq_evaluate_query_recall(query => $$
 $$, exact_search => true);
 ```
 
+## Query sampling <badge type="tip" text="since v0.5.2" />
+
+You may find it more useful to evaluate user-generated queries than randomly generated ones. VectorChord includes a built-in method to sample and store recent user queries. These records are grouped by index, making them easily accessible and seamlessly integrable with recall evaluation.
+
+```sql
+-- Enable the query sampling
+ALTER SYSTEM SET vchordrq.query_sampling_enable = on;
+
+-- Set the maximum number of queries to record per index
+ALTER SYSTEM SET vchordrq.query_sampling_max_records = 1000;
+
+-- Set the sampling rate (e.g., 1.0 for all queries, 0.01 for 1%)
+-- For high-traffic systems, a rate of 0.01 or lower is recommended.
+ALTER SYSTEM SET vchordrq.query_sampling_rate = 1;
+SELECT pg_reload_conf();
+
+-- Now, execute an example vector search
+-- SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 10;
+```
+
+After queries have been executed and sampled, you can inspect the captured components of them, including the schema, index, table, column, operator, and vector.
+
+```sql
+-- List recorded queries for a specific index
+SELECT * from vchordrq_sampled_queries('idx');
+-- schema_name | index_name | table_name | column_name | operator |    value
+---------------+------------+------------+-------------+----------+--------------
+-- public      | idx        | items      | embedding   | <->      | [0.5,0.25,1]
+
+-- List recorded queries for all accessible indexes
+-- DANGER: The result may unexpectedly mix records from multiple indexes. Please check carefully before use. // [!code error]
+-- SELECT COUNT(DISTINCT index_name) FROM vchordrq_sampled_queries WHERE ... // [!code error]
+SELECT * from vchordrq_sampled_queries;
+```
+
+To measure the recall of the recorded queries, you can reconstruct the target query statement. The `vchordrq_sampled_queries` view is **not** recommended here because the evaluated recall will be incorrect if the records come from more than one index.
```

Comment:
1. It's not `reconstruct`.
2. The text above doesn't even mention that `vchordrq_sampled_queries` is a view.
3. It's not recommended, it's strictly forbidden.